core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -573,11 +573,6 @@ angular.module('oppia').directive('conversationSkin', [
                has to be reset each time a new card is loaded to prevent
                unwanted scrolling in the new card. */
             $location.hash(null);
-            $scope.pendingCardWasSeenBefore = false;
-            if (PlayerTranscriptService.hasEncounteredStateBefore(
```

Comment:
This same check is present after an answer is submitted which is the only location where it should be present, hence this is removed.